components/value-distribution/src/main/java/org/datacleaner/result/ValueCountingAnalyzerResult.java

```diff
@@ -61,7 +61,7 @@
 
     public Integer getUnexpectedValueCount();
 
-    public boolean hasAnnotatedRows(String value);
+    public Boolean hasAnnotatedRows(String value);
```

Comment:
Right now any calls like this will start throwing exceptions:

```
if (result.hasAnnotatedRows("foo")) {
 ...
}
```

We might better then rename the method so that we can at least catch it at compile time. But in general I don't care about trying to patch the interface here - it's a beast that eventually needs to be redone, but a lot of consideration needs to go into it because at the same time the analyzer has to return some thing from which metrics can automatically be gathered.